recipes/capstone/all/conanfile.py

```diff
@@ -35,21 +36,19 @@ class CapstoneConan(ConanFile):
         "fPIC": True,
         "use_default_alloc": True,
     }
+    implements = ["auto_shared_fpic"]
 
     _archs = ["arm", "m68k", "mips", "ppc", "sparc", "sysz", "xcore", "x86", "tms320c64x", "m680x", "evm"]
     options.update({a: [True, False] for a in _archs})
     default_options.update({a: True for a in _archs})
 
+    def _is_universal2(self, info=False):
+        return Version(self.version) >= "5.0.3" and is_apple_os(self)
+
     def export_sources(self):
         export_conandata_patches(self)
 
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
     def configure(self):
-        if self.options.shared:
-            self.options.rm_safe("fPIC")
         self.settings.rm_safe("compiler.cppstd")
         self.settings.rm_safe("compiler.libcxx")
```

Comment:
Should I return this?
```py
def configure(self):
    if self.options.shared:
        self.options.rm_safe("fPIC")
    self.settings.rm_safe("compiler.cppstd")
    self.settings.rm_safe("compiler.libcxx")
```

Reply:
yes, this is necessary Conan-wise, thanks!

Reply:
@AbrilRBS ty, wht bout linter...